pkg/infra/kubernetes/exec_unit_test.go

```diff
@@ -279,7 +279,7 @@ func Test_transformPod(t *testing.T) {
                         image: '{{ .Values.testUnitImage }}'
                         name: web
                         resources: {}
-                      serviceAccountName: testUnit
+                      serviceAccountName: testnit
```

Comment:
minor/funny, but did you mean to write `testunit` (with a "u")? It sounds like this is a test nit.

(Is this comment a nitnit? a nit<sup>2</sup>?)